libyul/Scope.h

```diff
@@ -25,8 +25,6 @@
 
 #include <libyul/YulString.h>
 
-#include <libsolutil/Visitor.h>
-
```

Comment:
This is somewhat unrelated to the rest, but this include had no business being here.